proto-google-cloud-asset-v1p1beta1/src/main/proto/google/cloud/asset/v1p1beta1/asset_service.proto

```diff
@@ -0,0 +1,230 @@
+// Copyright 2019 Google LLC.
```

Comment:
2020 :)

Reply:
This one is the raw proto copied from the source at https://github.com/googleapis/googleapis/blob/master/google/cloud/asset/v1p1beta1/asset_service.proto